Dockerfile

```diff
@@ -33,6 +33,7 @@ ENV USER=$UNAME
 ENV HOME=/home/$USER
 ENV OUTPATH=$HOME/out/
 ENV PBPATH=$HOME/privacybadger/
+ENV EXTENSIONS=$HOME/parallel-extensions/
```

Comment:
Should we add a note to the README somewhere noting how to load another extension alongside Privacy Badger when running with Docker?

Reply:
Should we add any zip and CRX files inside `parallel-extensions` to `.gitignore`?

Reply:
added a note in the `2. Run A Scan` section of the README :+1:

Reply:
also added all zipped and CRX files in `parallel-extensions/` to the gitignore